external-crates/move/crates/move-vm-runtime/src/jit/execution/ast.rs

```diff
@@ -108,13 +108,14 @@ pub struct Module {
     pub instantiation_signatures: SignatureCache,
 
     /// constant references carry an index into a global vector of values.
-    pub constants: Vec<Constant>,
+    pub constants: ConstantCache,
 }
 
 pub type SignatureCache = BTreeMap<SignatureIndex, VMPointer<Vec<Type>>>;
+pub type ConstantCache = BTreeMap<ConstantPoolIndex, VMPointer<Constant>>;
```

Comment:
I'm guessing this is a `BTreeMap` as opposed to a `Vec<VMPointer<Constant>>` due to constant inlining? If so might be worth a comment.

Reply:
This change happens in the next PR.